make.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+# Copyright (C) 2016 the V8 project authors. All rights reserved.
+# This code is governed by the BSD license found in the LICENSE file.
+
+import os, shutil, subprocess, sys
+
+OUT_DIR = os.environ.get('OUT_DIR') or 'test'
+SRC_DIR = os.environ.get('SRC_DIR') or 'src'
+UPSTREAM = os.environ.get('UPSTREAM') or 'git@github.com:tc39/test262.git'
+MAINTAINER = os.environ.get('MAINTAINER') or 'goyakin@microsoft.com'
```

Comment:
I just sent an email to @ecmageneva and @bterlson requesting an email acc for test262 @ ECMA